sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -40,13 +42,14 @@ def stream_compressed_header_error():
 
 
 def stream_compressed_no_header():
-    with gzip.open("test.tar.gz", "wb") as f:
+    file_name = f"{str(uuid.uuid4())}.tar.gz"
```

Comment:
Maybe just use tempfile?

Reply:
Updated to just reuse the existing function you linked since they both give gzip byte stream